src/components/helpers/Overlay/style.scss

```diff
@@ -30,8 +30,5 @@
 
 .Overlay-open {
   overflow: hidden;
-  position: fixed;
-  top: 0;
-  bottom: 0;
```

Comment:
Will removing these work? How does the overlay position itself in front of the content now?